sdk/core/core-amqp/src/ConnectionContextBase.ts

```diff
@@ -144,13 +144,14 @@ export const ConnectionContextBase = {
       (!isNode && typeof window !== "undefined" && (window as any).WebSocket)
     ) {
       const socket = parameters.config.webSocket || (window as any).WebSocket;
-      const host = parameters.config.host;
+      const host = connectionOptions.host;
```

Comment:
No defaulting to the parameters.config.host here? Or is it assumed it'll be set in connectionOptions (ie, the default value setting is done elsewhere?)

Reply:
Ah, this change isn't really needed. Yes, connectionOptions.host should always be specified, but so should parameters.config.host.

Reply:
Or should this be 'hostname'?

Reply:
I would recommend to revert this change. Everywhere else inside this if block we use `parameters.config`, but here we use the connection options. It is not clear to me why we would do that.

Reply:
Reverted!